Sources/Actions/Profile/GroupMembership.php

```diff
@@ -120,6 +120,10 @@ public function show(): void
 		);
 		$open_requests = Db::$db->fetch_all($request);
 		Db::$db->free_result($request);
+		
+		$open_requests = array_map(function($v) {
```

Comment:
I could be blind and if so please forgive me. It appears $v is undefined here?

Reply:
$v will be the value of the array.  Its a callback.

Reply:
Changed it to $request so its more clear?